Assets/Scripts/Plants/Plant.cs

```diff
@@ -80,6 +85,21 @@ public class Plant : NetworkBehaviour
         /// </summary>
         private float elapsedSinceProduced;
 
+        /// <summary>
+        ///  Seed Particle System
+        /// </summary>
+        public ParticleSystem seedParticle;
```

Comment:
Make this a scriptable object similar to ResourceIcons instead of different fields for each